compliance/github/empty_repository/empty_repository.pt

```diff
@@ -0,0 +1,137 @@
+name "GitHub.com Empty Repository Report"
```

Comment:
would this be better titled "abandoned" repository? seems like it's checking for activity, not content ("empty")?

Reply:
Renamed to `GitHub.com Unpermitted Sized Repositories` and modified description in commit b7dba63